<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>983</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="minimumSize">
            <size>
             <width>157</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>157</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QFrame" name="frame_10">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QFrame" name="frame_11">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" rowspan="4" colspan="4">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame_22">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>100</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_22">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>551</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QFrame" name="frame_12">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>77</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>77</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QFrame" name="frame_13">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>77</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>77</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QFrame" name="frame_14">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QFrame" name="frame_15">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QFrame" name="frame_20">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_20">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QFrame" name="frame_19">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_19">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QFrame" name="frame_18">
           <property name="minimumSize">
            <size>
             <width>157</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>157</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_18">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QFrame" name="frame_17">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="5">
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QFrame" name="frame_16">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>78</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_16">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>81</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_22">
       <item>
        <widget class="QFrame" name="frame_21">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>821</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
